Libraries/Utilities/Platform.ios.js

```diff
@@ -12,8 +12,21 @@
 
 'use strict';
 
+var invariant = require('invariant');
+
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => {
+    invariant(
+      obj && typeof obj === 'object',
+      'Platform.select: Must be called with an object'
+    );
+    invariant(
```

Comment:
That's true, however the good thing is it can be used with non-style values and also at any level of StyleSheet object, so it's likely to cause some errors, when you write:
```
Stylesheet.create({ ...Platform.select({ ios: {} }) })
```
vs
```
StyleSheet.create({ container: { marginTop: Platform.select({}) } })
```